test/proxy.py

```diff
@@ -3,26 +3,34 @@
 from starkware.starknet.testing.starknet import Starknet
 from starkware.starknet.definitions.error_codes import StarknetErrorCode
 from utils.Signer import Signer
-from utils.utilities import compile, cached_contract, assert_revert, assert_event_emmited
-from utils.TransactionSender import TransactionSender
+from utils.utilities import compile, cached_contract, assert_revert, assert_event_emmited, get_execute_data
+from utils.TransactionSender import TransactionSender, from_call_to_call_array
+from typing import Optional, List, Tuple
+
 from starkware.starknet.compiler.compile import get_selector_from_name
 
 signer = Signer(1)
 guardian = Signer(2)
 wrong_signer = Signer(3)
 wrong_guardian = Signer(4)
 
```

Comment:
You could define a type for calls, either here or in a shared file:

```python
Call = Tuple[str, str, List]
```

Reply:
i agree with all of the comments, we had to merge this PR a bit quicker but i will implement the changes next week 👍